### ISSUE TITLE
Loss Nan -- On phoenix dataset while training

### ISSUE THREAD

**Issue by pallapumohankrishna** [hulianyuyy/CorrNet_Plus#14]
Dear @hulianyuyy 

I am training Phoenix2014 dataset, but i can see Nan loss. Why is it coming : Please find the below attachment.

<img width="1411" height="599" alt="Image" src="https://github.com/user-attachments/assets/763218f5-143e-4815-b3cf-c73ff24bdd22" />

**Comment by hulianyuyy** [comment]
This is normal. For some videos, the video sequence length after pooling is smaller than the gloss sequence, thus raising the warning.

**Comment by pallapumohankrishna** [comment]
@hulianyuyy ,

I have been observing from past 5 epochs, loss is Nan and if we observe the log file which i have attached, no significant changes over 5 epochs. I'm confused about seeing the results. Could you please help me in this issue?

<img width="1385" height="992" alt="Image" src="https://github.com/user-attachments/assets/6b76573d-51c8-4b97-bddf-e8f508738057" />

**Comment by pallapumohankrishna** [comment]
Dear @hulianyuyy,

Please help what is the reason for getting nan loss every time. 

I followed all the steps mentioned in the repository. Just installed recent versions of respective packages.

**Comment by hulianyuyy** [comment]
I suspect that you don't correctly install the CTC decode or sclite package. You can directly perform inference with the pretrained weight for CorrNet to check whether you can get the correct result. You can check whether it report errors during inference.
